vendor/src/github.com/docker/engine-api/types/auth.go

```diff
@@ -5,7 +5,6 @@ type AuthConfig struct {
 	Username      string `json:"username,omitempty"`
 	Password      string `json:"password,omitempty"`
 	Auth          string `json:"auth,omitempty"`
-	Email         string `json:"email"`
```

Comment:
@dmcgowan I think you are right the omitempty flag is probably a better approach to support backwards compatibility.